packages/manager/modules/nutanix/package.json

```diff
@@ -12,16 +12,19 @@
   "author": "OVH SAS",
   "main": "./src/index.js",
   "dependencies": {
+    "@ovh-ux/manager-models": "^1.4.3",
     "bootstrap4": "twbs/bootstrap#v4.0.0",
     "lodash": "^4.17.14"
   },
   "peerDependencies": {
+    "@ovh-ux/manager-billing-components": "^2.0.0 || ^3.0.0",
```

Comment:
Following module has `moment` has peerDependency which means that the standalone
application `@ovh-ux/manager-nutanix-app` must have it as a `dependency`.